src/Sulu/Bundle/MediaBundle/Tests/Unit/DependencyInjection/ImageTransformationCompilerPassTest.php

```diff
@@ -27,10 +27,7 @@ protected function registerCompilerPass(ContainerBuilder $container): void
         $container->addCompilerPass(new ImageTransformationCompilerPass());
     }
 
-    /**
-     * @test
-     */
-    public function ifCompilerPassCollectsServicesByAddingMethodCallsTheseWillExist(): void
+    public function testCollectingTaggedServices(): void
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes